From 1663d9902b9985a8bad13918fa603c759aaaaefc Mon Sep 17 00:00:00 2001
From: Eric Sandall <sandalle@sourcemage.org>
Date: Thu, 13 Sep 2018 16:14:41 -0700
Subject: [PATCH 09/23] Add missing <sys/sysmacros.h> include

For major & minor macros in mountpoint.c.

[ismael@sourcemage.org: Added proper commit message]

Signed-off-by: Ismael Luceno <ismael@iodev.co.uk>
---
 sysvtools/mountpoint.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/sysvtools/mountpoint.c b/sysvtools/mountpoint.c
index c2477279ce7d..02d30805a4dd 100644
--- a/sysvtools/mountpoint.c
+++ b/sysvtools/mountpoint.c
@@ -14,6 +14,7 @@
  *		2 of the License, or (at your option) any later version.
  */
 
+#include <sys/sysmacros.h>
 #include <sys/stat.h>
 #include <unistd.h>
 #include <stdlib.h>
-- 
2.44.0

